src/app/dashboard/Missions/ListItem.jsx

```diff
@@ -0,0 +1,196 @@
+import { Box, Grid } from "@material-ui/core";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import { makeStyles } from "@material-ui/core/styles";
+import ArrowRightAltIcon from "@material-ui/icons/ArrowRightAlt";
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import PanToolIcon from "@material-ui/icons/PanTool";
+import React from "react";
+import styled from "styled-components";
+
+import { Button } from "../../component";
+import Mission from "../../model/Mission";
+import _ from "../../utils/lodash";
+
+const useStyles = makeStyles((theme) => ({
+  row: {
+    flexWrap: "nowrap",
+    overflow: "hidden",
+  },
+  asap: {
+    backgroundColor: theme.color.red,
+    color: theme.color.white,
+  },
+  formControl: {
+    width: "100%",
+  },
+  select: {
+    width: "100px",
+  },
+}));
+
+const ToMissionDetails = styled.div`
+  position: absolute;
+  right: 0px;
+  top: 0px;
+`;
+const Root = styled(Box)`
+  position: relative;
+`;
+
+/** BEGIN ACTION*/
+const UnasignedStatus = ({ missionId }) => {
+  const classes = useStyles();
+  const handleChange = (event) => {
+    event.preventDefault();
+    Mission.update(missionId, {
+      fundedStatus: event.target.value,
+      status: Mission.Status.tentative,
+    });
+  };
+  const options = [
```

Comment:
Can move this outside the component since it's static.
Otherwise it instantiates this object every time this component renders